services/src/main/java/za/co/sfy/lucid/sams/rest/vo/data/writer/TopicsRequests.java

```diff
@@ -0,0 +1,107 @@
+
+package za.co.sfy.lucid.sams.rest.vo.data.writer;
+
+import java.io.Serializable;
+
+
+
+public class TopicsRequests implements Serializable {
+
+    private static final long serialVersionUID = 6627252016141632712L;
+
+    private Integer maintopicid;
+    private Integer mainposition;
+    private Short description;
+    private Short afrdescription;
+    private Short verndescription;
```

Comment:
Not sure if requests work with Short maybe it does but you may have to change this to Integer and then convert to Short in the corresponding Mapper class, remember to do a null check before you attempt conversion